python/pyspark/ml/deepspeed/tests/test_deepspeed_distributor.py

```diff
@@ -227,7 +227,7 @@ def setUpClass(cls) -> None:
             conf = conf.set(k, v)
         conf = conf.set(
             "spark.worker.resource.gpu.discoveryScript", cls.gpu_discovery_script_file_name
-        )
+        ).set("spark.python.unix.domain.socket.enabled", "false")
```

Comment:
Does this mean `deepspeed` doesn't support UDS?

Reply:
It does .. I think the problem is that the temporary directory (`spark.python.unix.domain.socket.dir`) isn't properly set through somehow .. I believe it's more a test-specific issue.